src/main/java/uk/gov/hmcts/cmc/claimstore/documents/DefendantResponseReceiptService.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.cmc.claimstore.documents;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.config.properties.pdf.DocumentTemplates;
+import uk.gov.hmcts.cmc.claimstore.documents.content.DefendantResponseReceiptContentProvider;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.reform.cmc.pdf.service.client.PDFServiceClient;
+
+import static java.util.Objects.requireNonNull;
+
+@Service
+public class DefendantResponseReceiptService {
+
+    private final DefendantResponseReceiptContentProvider contentProvider;
+    private final DocumentTemplates documentTemplates;
+    private final PDFServiceClient pdfServiceClient;
+
+    @Autowired
+    public DefendantResponseReceiptService(
+        DefendantResponseReceiptContentProvider contentProvider,
+        DocumentTemplates documentTemplates,
+        PDFServiceClient pdfServiceClient
+    ) {
+        this.contentProvider = contentProvider;
+        this.documentTemplates = documentTemplates;
+        this.pdfServiceClient = pdfServiceClient;
+    }
+
+    public byte[] createPdf(Claim claim) {
```

Comment:
Final modifier?

Reply:
Myself get annoyed with all these finals flying around. What about removing these and forbidding parameter assignment with checkstyle?

http://checkstyle.sourceforge.net/config_coding.html#ParameterAssignment

@timja-kainos, @rlewan, @radoslawn-kainos? ☝️

Reply:
```
[ant:checkstyle] [WARN] /Users/damiand/Projects/MoJ/CMC/claim-store/src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/SealedClaimContentProvider.java:37:15: Assignment of parameter 'claim' is not allowed. [ParameterAssignment]
```

Sample error when module is enabled and rule violated.